labelbox/client.py

```diff
@@ -71,10 +72,9 @@ def __init__(self,
             'X-User-Agent': f'python-sdk {SDK_VERSION}'
         }
 
-    #TODO: Add exponential backoff so we don'tt overwhelm the api
     @retry.Retry(predicate=retry.if_exception_type(
         labelbox.exceptions.InternalServerError))
-    def execute(self, query, params=None, timeout=30.0):
+    def execute(self, query, params=None, timeout=30.0, experimental=False):
```

Comment:
Do you think this should happen at the function level or when a user init's their SDK?

What If I call execute a bunch of times and each time I have to remember to pass in experimental? Feel like it will make my code more messy as opposed to opting into it once

That will also let you just change the graphql endpoint exactly one

Reply:
this is all being managed by the `beta` decorator. Adding here for context.